templates/geosafe/analysis/create.html

```diff
@@ -492,14 +474,33 @@ <h4 class="modal-title" id="myModalLabel">{% trans "Upgrade Your Browser" %}</h4
     {% include  "geosafe/analysis/dynamic_scripts/create/main.html" %}
 
     <script type="text/javascript">
+    window.onload = function() {
+        //page fully loaded, hide the slow-loading notification
+        $(".slow-loading").toggle(false);
+    };
         {# Initialization script of the page. #}
         $(document).ready(function () {
+            $(".slow-loading").toggle(true);
 
             //check browser compatibility against ECMA Script 5
             if (!Modernizr.json || !Modernizr.es5) {
                 $('#upgrade-browser').modal('toggle');
             }
 
+            // Legend
+            $(".legend").toggle(is_legend);
+            $("#legend-exposure-body").toggle(is_exposure);
+            $("#legend-hazard-body").toggle(is_hazard);
+            $("#legend-impact-body").toggle(is_impact);
+            //hide/view legend
+            $("#legend-title").click(function () {
+                $("#legend-body").toggle();
+            });
+
+            // Add scrollbar in case of overflowing in dimension-y
+            $("#legend-body").css("max-height", 0.7 * w_height);
+            $("#legend-body").css("overflow-y", "auto")
```

Comment:
i think this one is from another issue.
i'll separate them when doing another ticket.